src/engines/synths/nuke/nuke.cpp

```diff
@@ -95,6 +96,7 @@ namespace otto::engines {
       case ui::Rotary::Green: return knobs[1].rotation;
       case ui::Rotary::White: return knobs[2].rotation;
       case ui::Rotary::Red: return knobs[3].rotation;
+      default: throw util::exception("Unknown rotary value");
```

Comment:
we should not add a default when all enum options are exhausted, especially not scoped enums

Reply:
Switched to a pragma to quiet the compiler.